CMakeLists.txt

```diff
@@ -19,6 +19,17 @@ add_executable(
   utils.cc
   )
 
+add_executable(
```

Comment:
I believe this makes the build slower. Let me try improving this in a separate PR